src/main/java/com/restaurant/service/impl/RatingServiceImpl.java

```diff
@@ -32,12 +38,16 @@ public class RatingServiceImpl implements RatingService {
 	 * @see com.restaurant.dto.RatingDto
 	 */
 	@Override
-	public RatingDto createRating(RatingDto ratingDto, long id) {// rating dto
+	public RatingDto createRating(RatingDto ratingDto, long orderId,long userId) {// rating dto
```

Comment:
update comment details for the method

Reply:
Done

Reply:
add orderId and userId param in method comment